src/playback-watcher.js

```diff
@@ -51,7 +51,9 @@ export default class PlaybackWatcher {
     this.tech_.on('seekablechanged', fixesBadSeeksHandler);
     this.tech_.on('waiting', waitingHandler);
     this.tech_.on(timerCancelEvents, cancelTimerHandler);
-    this.monitorCurrentTime_();
+    if (this.tech_.hasStarted_) {
```

Comment:
I don't think so. I think we would have to wait for either the `ready` or `canplay` event from the tech